app/js/components/manageApps/ManageApps.jsx

```diff
@@ -362,8 +362,11 @@ export default class ManageApps extends React.Component {
     });
   }
 
-  handleUserClick(name) {
-    toastr.info(`Sorry, there is no open web app for ${name}`);
+  handleUserClick(appType, name) {
+    appType === "OWA"?
```

Comment:
What happens if it is a module?

Reply:
The else condition occurs. OWA's are the only ones that need a message informing the user that they need to install the addon in order to link to the apps page. Otherwise the rest will fall under the condition of them not having Open Web Apps.